scripts/cnv_wdl/cnv_common_tasks.wdl

```diff
@@ -87,11 +87,14 @@ task CollectCounts {
     File? gatk4_jar_override
 
     # Runtime parameters
-    Int? mem
+    Int? mem_gb
     String gatk_docker
     Int? preemptible_attempts
     Int? disk_space_gb
 
+    Int machine_mem_mb = select_first([mem_gb  * 1000, 7500])
+    Int command_mem_mb = machine_mem_mb - 1000
```

Comment:
Why don't you do this in the tasks above?

Reply:
Thanks for catching this.  Actually, I think this makes the tests fail---you can't perform the multiply operation inside select_first, apparently.